genGo.go

```diff
@@ -224,6 +227,9 @@ func (gen *CodeGenerator) GoComplexType(v *ComplexType) {
 			if fieldType == "time.Time" {
 				gen.ImportTime = true
 			}
+			if element.Optional && !strings.HasPrefix(fieldType, `*`) {
```

Comment:
I noticed that this change include changes in PR #96, does this depends on that PR checkin first?